Catappult/appcoins-sdk
# Implement First Web Payment

**What does this PR do?**

This PR implements the feature of the First Web Payment using the URL given by the PayFlowPriority response.

It focuses in 2 types of communication with the Web Page:
1. DeepLink - used to open the Application from the Browser
2. WebSocket - used to receive information from the Browser regarding the status of the Purchase

**How should this be manually tested?**

Use this version in the Diceroll app and apply the necessary changes. Remove the wallet or the gameshub from the device. Make a payment via SDK which will prompt to use the Browser to make the payment. Finish the payment and go back to the game. The result should be successful.

**What are the relevant tickets?**

Tickets related to this pull-request: 
- [APT-3949](https://aptoide.atlassian.net/browse/APT-3949)
- [APT-3969](https://aptoide.atlassian.net/browse/APT-3969)

**Questions:**

This feature adds a new dependency: `org.java-websocket:Java-WebSocket:1.5.3`

**Code Review Checklist**
